### PR TITLE
Add crate *version* to a template generator

### PR DESCRIPTION
**Pull request by mexus**
I really miss this functionality sometimes.. and so do some other people :)

While preparing the PR I've tried to follow the library's code style, but I definitely think it could use some `cargo fmt` ;)

### CODE REVIEW

**Comment by mexus** [comment]
@livioribeiro ping?..

**Comment by livioribeiro** [comment]
Oh, sorry, it seems the notifications sometimes don't reach me.

Could you update the doc comments in `src/lib.rs` (and then regenerate the README.md) and adding an integration test?

**Comment by mexus** [comment]
Sure! Will do

**Comment by mexus** [comment]
@livioribeiro  I hope I've got your point :) Made an additional commit, please let me know if you want them squashed.

Also pls tell me if you don't want me to ping you :)

File: src/lib.rs [hunk 1]
```diff
@@ -101,7 +101,7 @@
 //!
 //! {{readme}}
 //!
-//! Some additional info here
+//! Some additional info here, like: "current version is {{version}}"
```
Comment (livioribeiro) [inline src/lib.rs]: Could you the `{{version}}` placeholder right below the `{{readme}}`? Like this:

```
//! {{readme}}
//!
//! Current version: {{version}}
```

And there is also the rendered example in the fenced block right below that.

**Comment by livioribeiro** [comment]
Just a minor change and it is good to go.

> Also pls tell me if you don't want me to ping you :)

That's ok, no problem :)

File: src/lib.rs (hunk 1 above)
Comment (mexus) [reply src/lib.rs]: np :)

**Comment by mexus** [comment]
Done!

If everything's all right, do you want me to squash all the commits together?

**Comment by livioribeiro** [comment]
> If everything's all right, do you want me to squash all the commits together?

Doesn't github do that when merging the pull request?

**Comment by livioribeiro** [comment]
Thanks a lot! 😃

**Comment by mexus** [comment]
Thank you for merging the request so fast! :)

(github does squashing, but if I do that manually I would remove all the *extra* commit messages)

**Comment by inv2004** [comment]
@livioribeiro 
Could you please publish new version?

**Pull request merged**